force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +103,13 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        if(a == null){
+            return null;
+        }
+        else if(Math.Mod(a, 2) == 0){
+            return 'Even';
+        }
+        return 'Odd'; // Replace null with the variable you used to store the result
```

Comment:
Nice work. Can you do it with a ternary operator? That is what I like for these kind of problems.